camelcase-keys/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for camelcase-keys 4.0
+// Project: https://github.com/sindresorhus/camelcase-keys#readme
+// Definitions by: Mohamed Hegazy <https://github.com/mhegazy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = camelcase_keys;
+declare function camelcase_keys(input: any, options?: { exclude?: string[] | RegExp[], deep?: boolean }): any;
```

Comment:
well.. this assumes that that the object is homogeneous. which might not be correct.